packages/prometheus/data_stream/collector/manifest.yml

```
@@ -169,6 +169,15 @@ streams:
         default: prometheus.collector
         required: true
         show_user: true
+      - name: metrics_count
+        type: bool
+        title: Metrics Count
+        multi: false
+        required: false
+        show_user: false
+        default: false
+        description: >
+          Adds a `metrics_count` field to the document. This field indicates the total number of Prometheus metrics contained within the Elasticsearch document.
```

Its a boolean config. When I read it for the first time, it felt like I have to add a field in the config.
```suggestion
          Enable the `metrics_count` to get the total number of Prometheus metrics contained within the Elasticsearch document
```